build/charts/minio/values.yaml

```diff
@@ -283,6 +283,20 @@ policies:
         - "s3:DeleteObject"
         - "s3:GetBucketLocation"
 
+- name: laf_owner_readonly_by_prefix
```

Comment:
To support limiting resource usage, you need to: 
- create a new minio group `laf_owner_readonly_by_prefix_group`
- create a new minio policy `laf_owner_readonly_by_prefix`